### ISSUE TITLE
When there is no ambiguity, jump to match by pressing enter

### ISSUE THREAD

**Issue by nilscox** [usernamehw/vscode-find-jump#21]
Hi,

When there is only one matching text left, it would be great to jump to this location by pressing enter.

I can write a PR if you want, do you have any contributing guidelines?

**Comment by usernamehw** [comment]
For that case you can try to create command `findJump.goToFirstMatch` without assigning keyboard shortcut by default.

**Comment by nilscox** [comment]
Just to make sure I understand, you want another command that, when executed, will activate find => jump and move the cursor to the first match when pressing enter, is that right?

I suggest a slightly different behavior, which would be to keep the same command (`findJump.activate`), but when you type enough letters to have only a single match and you press enter, the cursor will move there.
This can be disabled by default, and enabled with a config like `findJump.goToSingleMatch` for example.

**Comment by usernamehw** [comment]
No. What I mean is - create a command `findJump.goToFirstMatch` that would only work when extension is activated (after `findJump.activate`).

Users will then have to manually assign the keybinding, like:

```js
{
	"key": "enter",
	"command": "findJump.goToFirstMatch",
	"when": "findJumpActive && editorTextFocus"
},
```

**Comment by nilscox** [comment]
Oh yes, sure. That's great.

**Comment by usernamehw** [comment]
Fixed by #22
